### PR TITLE
Change heading to Zebra in README.md

### PR DESCRIPTION
**Pull request by sandakersmann**
Change heading to Zebra in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# The Zcash Foundation Zcash client.
+# Zebra
```
Comment (dconnolly) [inline README.md]:
```suggestion
# Zebra: the Zcash Foundation client.
```

**Review by dconnolly — CHANGES_REQUESTED** [review]
Made a small suggestion, I think you need to accept since I can't push to your fork.

**Comment by sandakersmann** [comment]
Done :)

**Review by dconnolly — APPROVED** [review]
🙌

**Pull request merged**